### ISSUE TITLE
🤔[question] How to seup trail filter?

### ISSUE THREAD

**Issue by scotthuang1989** [determined-ai/determined#9907]
### Describe your question

I try to use this Filter to filter some trails from a experiments.

But I found  2 issues
1. some  filter option is from other experiments. 
2. not all  filter options  of this experiments is listed at this place.

![image](https://github.com/user-attachments/assets/1f4ab353-e14b-4780-b01d-e906f29e9223)

### Checklist

- [X] Did you search [the docs](https://docs.determined.ai/latest/) for a solution?
- [X] Did you search github issues to find if somebody asked this question before?

**Comment by EmilyBonar** [comment]
Hi @scotthuang1989, I'm a frontend developer and would be happy to try and help you with this. First off I need to collect some information.

1. What version of the software are you using? This should be in the lower left hand corner of the page, ie `0.36.1`.
2. What is the URL where you are seeing this problem?
3. What filter options are you expecting to see that are missing?

**Comment by scotthuang1989** [comment]
Hi Emily,

version is  0.35.0

url is: 127.0.0.1:8080/det/searches/131/runs

my expection is
1.  all hyper parameters that is searched   should appear as a filter condition.  for example,  in this experiment , I  submit a grid search over 2  optimizer and 3 learning rate.  I want filter by optimizer,  to see training result of  different learning rate with this optimizer.
2.  hyper parameter from other experiments should not appeared as a filter condition.

**Comment by EmilyBonar** [comment]
1. Ok, good to know. Can you list for me the names, data types (text, number, array, etc), and whether or not you can filter on them for each of those 5 hyperparameters?
2. This is a known issue, and I agree that it's annoying. The searches and runs view is still a new feature, and hopefully we will be able to solve this in the near future.
